woocommerce/woocommerce-ios
# Update the empty state image in the shipping class list selector

Fixes #1653 

## Changes

- Added an `UIImage` extension for the shipping class list selector empty state and showed it via the view properties

## Testing

Prerequisite: the store has no shipping classes, or hard code to always show the no results state

- Launch the app
- Go to Products tab
- Tap on a Product
- Tap "Edit"
- Tap on the shipping settings row
- Tap the Shipping class row --> the empty state should look as expected

## Example screenshots

Light mode | Dark mode
-- | --
![Simulator Screen Shot - iPhone 8 - 2019-12-31 at 11 35 32](https://user-images.githubusercontent.com/1945542/71609274-1d184100-2bc2-11ea-8864-972656fb8be5.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-31 at 11 35 15](https://user-images.githubusercontent.com/1945542/71609273-1c7faa80-2bc2-11ea-88ff-04badfc03cfd.png)


Update release notes:
